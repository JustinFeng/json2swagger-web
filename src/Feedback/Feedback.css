.App-feedback {
  margin-top: 30px;
  text-align: left;
}