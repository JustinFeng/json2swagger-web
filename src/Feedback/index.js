import React from 'react';
import './Feedback.css';

const Feedback = () =>
  <div className="App-feedback">
    <a href="https://github.com/JustinFeng/json2swagger-web/issues/new" rel="noopener noreferrer"
       target="_blank">
      Report a bug / Feedback
    </a>
  </div>;

export default Feedback;
