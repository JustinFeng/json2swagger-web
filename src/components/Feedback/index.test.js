import React from 'react';
import renderer from 'react-test-renderer';
import Feedback from './index';

describe('Feedback', () => {
  it('renders correctly', () => {
    const app = renderer
      .create(<Feedback />)
      .toJSON();

    expect(app).toMatchSnapshot();
  });
});
