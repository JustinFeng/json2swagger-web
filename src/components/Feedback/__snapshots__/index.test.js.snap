// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feedback renders correctly 1`] = `
<div
  className="App-feedback"
>
  <a
    href="https://github.com/JustinFeng/json2swagger-web/issues/new"
    rel="noopener noreferrer"
    target="_blank"
  >
    Report a bug / Feedback
  </a>
</div>
`;
